src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -205,6 +225,62 @@ protected function isAttributeToNormalize($object, $attributeName, &$context)
     }
 
     /**
+     * Validates the submitted data and denormalizes it..
+     *
+     * @param string      $currentClass
+     * @param string      $attribute
+     * @param mixed       $data
+     * @param string|null $format
+     * @param array       $context
+     *
+     * @return mixed
+     *
+     * @throws UnexpectedValueException
+     * @throws LogicException
+     */
+    protected function validateAndDenormalize($currentClass, $attribute, $data, $format, array $context)
```

Comment:
After double checking, this extension point isn't needed anymore for API Platform. As I've no use case anymore, I changed the visibility of this method to private.